src/main/kotlin/org/rust/ide/colors/RsColors.kt

```diff
@@ -19,17 +19,18 @@ enum class RsColor(humanName: String, val default: TextAttributesKey? = null) {
     FIELD("Variables//Field", Default.INSTANCE_FIELD),
     CONSTANT("Variables//Constant", Default.CONSTANT),
 
-    FUNCTION("Functions//Function", Default.FUNCTION_DECLARATION),
-    METHOD("Functions//Method", Default.INSTANCE_METHOD),
-    ASSOC_FUNCTION("Functions//Associated function", Default.STATIC_METHOD),
+    FUNCTION("Functions//Function declaration", Default.FUNCTION_DECLARATION),
+    METHOD("Functions//Method declaration", Default.INSTANCE_METHOD),
+    ASSOC_FUNCTION("Functions//Associated function declaration", Default.STATIC_METHOD),
+    FUNCTION_CALL("Functions//Function and method call", Default.FUNCTION_CALL),
     MACRO("Functions//Macro", Default.IDENTIFIER),
 
     PARAMETER("Parameters//Parameter", Default.PARAMETER),
     MUT_PARAMETER("Parameters//Mutable parameter", Default.PARAMETER),
     SELF_PARAMETER("Parameters//Self parameter", Default.KEYWORD),
     LIFETIME("Parameters//Lifetime", Default.IDENTIFIER),
     TYPE_PARAMETER("Parameters//Type parameter", Default.IDENTIFIER),
-    CONST_PARAMETER("Parameters//Const parameter", CONSTANT.textAttributesKey),
+    CONST_PARAMETER("Parameters//Const parameter", Default.CONSTANT),
```

Comment:
Why is this changed?

Reply:
As we discussed in person, constant parameters are more parameters then constants (for example, in C++ there is a separate color for template params including constant params) so it's quite unclear why the color for them should depend on constant color.
Also, it can be confusing that `CONST_PARAMETER` changes when you change `CONSTANT` but not vice versa